### PR TITLE
Added method to print values of particle, and method to calculcate classical particle radius

### PR DESCRIPTION
**Pull request by sailew**
…assical particle radius.

## Description

Closes # .

## Checklist

Mandatory: 

- [x] I have added tests to cover my changes
- [x] All the tests are passing, including my new ones
- [x] I described my changes in this PR description

Optional:

- [x] The code I wrote follows good style practices (see [PEP 8](https://peps.python.org/pep-0008/) and [PEP 20](https://peps.python.org/pep-0020/)).
- [x] I have updated the docs in relation to my changes, if applicable
- [ ] I have tested also GPU contexts

### CODE REVIEW

File: xpart/particles/particles.py [hunk 1]
```diff
@@ -344,7 +344,7 @@ def classical_particle_radius(self):
         Method to calculate classical particle radius from particles
         """
         m0 = self.mass0*1.782661921e-36  # electron volt - kg conversion
```
Comment (kparasch) [inline xpart/particles/particles.py:346]: Perhaps we should use qe/c**2 to calculate the eV - kg conversion instead of hardcoding it.

Comment (sailew) [reply xpart/particles/particles.py]: Good idea, fixed it now!

**Comment by sailew** [comment]
Now tests have been added and checked for protons and ions in `tests/test_classical_particle_radius_and_value_print.py`, where also the printing of the particle values is tested.

**Review by giadarol — CHANGES_REQUESTED** [review]

File: xpart/particles/particles.py [hunk 2]
```diff
@@ -324,6 +325,28 @@ def to_pandas(self,
...
+        print('\n')
+
+    def classical_particle_radius0(self):
+        """ 
+        Method to calculate classical particle radius from reference particle
+        """
+        m0 = self.mass0*qe/(clight**2) # electron volt - kg conversion
+        r0 = (self.q0*qe)**2/(4*np.pi*epsilon_0*m0*clight**2)  #1.5347e-18 is default for protons
+        return r0
+
```
[the remark this answers is not in the document]
Comment (sailew) [reply xpart/particles/particles.py]: Updated now!

File: tests/test_classical_particle_radius_and_value_print.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        
+        particle_ref = xp.Particles.from_dict(input_data['particle'])
+        
+        
+        # Test the printing capture standard output by redirecting sys.stdout to a StringIO object
+        capturedOutput = io.StringIO()                  # Create StringIO object
+        sys.stdout = capturedOutput                     #  and redirect stdout.
+        particle_ref.print_values()
+        sys.stdout = sys.__stdout__                     # Reset redirect.
+        print('Captured', capturedOutput.getvalue())   # Now works as before.
```
Comment (giadarol) [inline tests/test_classical_particle_radius_and_value_print.py:75]: This test does not really check that the output is correct.

File: xpart/particles/particles.py [hunk 4]
```diff
@@ -324,6 +325,28 @@ def to_pandas(self,
         import pandas as pd
         return pd.DataFrame(dct)
 
+    def print_values(self):
+        """
```
Comment (giadarol) [inline xpart/particles/particles.py:329]: I would call the function "show(self)"

**Review by giadarol — CHANGES_REQUESTED** [review]

File: tests/test_classical_particle_radius_and_value_print.py (hunk 3 above)
Comment (sailew) [reply tests/test_classical_particle_radius_and_value_print.py:75]: Alright, is there any efficient way to check this or can we leave this method without tests?

Comment (sailew) [reply tests/test_classical_particle_radius_and_value_print.py]: Now added in test to show particles, no error should be raised if printed correctly.

**Review by giadarol — APPROVED** [review]

**Comment by szymonlopaciuk** [comment]
Merged in version 0.10.1

**Pull request closed without merging**